src/platform/mbed/net/src/net_if.cpp

```diff
@@ -140,4 +143,103 @@ char * mbed_inet_ntop(sa_family_t family, const void * src, char * dst, size_t s
 int mbed_inet_pton(sa_family_t family, const char * src, void * dst)
 {
     return 0;
+}
+
+int mbed_ioctl(int fd, unsigned long request, void * param)
+{
+    int ret = 0;
+    switch (request)
+    {
+    case SIOCGIFFLAGS: {
```

Comment:
Are `SIOCSIFFLAGS`, `SIOCGIFNAME` and `SIOCGIFINDEX` required too for CHIP ?

Reply:
For now, just only SIOCSIFFLAGS is used in CHIP